packages/core/src/language/nb.json

```diff
@@ -0,0 +1,49 @@
+{
+    "login": {
+        "logIn": "Logg inn",
+        "demo": "Demo"
+    },
+    "releaseNotes": {
+        "whatsNew": "Hva er nytt",
+        "releaseNotes": "Utgivelsesnotater"
+    },
+    "createIncident": {
+        "title": "Opprett sak i ServiceNow",
+        "info": "Vi samler inn informasjon om din enhet som en del av vår saksbehandlingsprosess. Ved å sende inn samtykker du i å dele følgende informasjon:",
+        "placeholderTitle": "Skriv inn sakstittel",
+        "placeholderDescription": "Skriv inn en komplett beskrivelse av saken din. Du trenger ikke oppgi informasjon om din brukerenhet.",
+        "created1": "Sak ",
+        "created2": " opprettet.",
+        "error": "En feil oppstod ved opprettelse av din sak: "
+    },
+    "information": {
+        "user": "Bruker",
+        "deviceBrand": "Enhetsmerke",
+        "device": "Enhet",
+        "OS": "Operativsystem",
+        "timezone": "Tidssone",
+        "locale": "Område"
+    },
+    "language": {
+        "language": "Språk",
+        "English": "Engelsk",
+        "Norwegian": "Norsk"
+    },
+    "settings": {
+        "title": "Innstillinger",
+        "aboutApp": "Om App'en",
```

Comment:
I know this is probably just copied from mad-expo-core but this apostrophe really irks me for some reason :S

Reply:
Yeah I'll remove it